app/assets/stylesheets/framework/variables.scss

```diff
@@ -0,0 +1,27 @@
+$breakpoint: 700px;
+
+$text: #444;
```

Comment:
This creates completely new variables which ignore all existing color and also doesn't work with the other themes.

Reply:
I wouldn't think the sign-up page would need to work with themes, because a not-yet-registered person won't have set a theme. But it certainly makes sense to use existing variables wherever possible, which will make changing the UI across the app easier in future.

Reply:
A podmin can set a default theme in the diaspora.toml (which I have done on my pod), and this should then still work for the registration page.

Reply:
Ah, thanks. I didn't realise that.